src/app.js

```diff
@@ -32,10 +43,14 @@ app.use('/api/v1/Generation', generationController);
 app.use('/api/v1/Auth', authController);
 app.use('/api/v1/Post', postController);
 
+// auth router attaches /login, /logout, and /callback routes to the baseURL
+app.use(auth(config));
+
 // TODO: delete this
 // dummy GET endpoint that returns Hello World
 app.get('/', (req, res) => {
```

Comment:
We can only send back response once, so maybe we could do the following instead to provide more info + context into what is happening: 

```javascript
app.get('/', (req, res) => {
  const userDetails = {
    idToken: req?.oidc?.idToken,
    accessToken: req?.oidc?.accessToken,
    refreshToken: req?.oidc?.refreshToken,
    idTokenClaims: req?.oidc?.idTokenClaims,
    user: req?.oidc?.user
  };
  res.send(req?.oidc?.isAuthenticated() ? userDetails : 'Logged out');
});
```

Reply:
Resolved